### ISSUE TITLE
Configuration Process Stuck at Verifying Git Repo Settings

### ISSUE THREAD

**Issue by SkyeRangerDelta** [Requarks/wiki-v1#191]
### Actual behavior
After configuring a new repository and manually editing parts of the config.yml, the configuration page of the "browser installer" sits on "Verifying Git repository settings..." with no apparent progress made. 

I've spent roughly 2 hours trying to work this out, including generating new SSH keys, saving them and changing the directory in the config, setting up the proper auth sections all in what I believe to be the proper fashion according to the documentation.

### Expected behavior
Progress on with the repository state.

### Steps to reproduce the behavior
Setup a private GitHub repository and initialize with readme.md. Continue through the configuration pages as per the "node wiki configure" command pages in the browser.

### Other Notes
I can only guess that I've just simply missed something in setup that wasn't apparent to me. I'm not a master at git, but I can work with it. Site Auth section has been filled with appropriate GitHub client information, Git Connection Info section is identical to the SSH section in the documentation except filled in with the "proper" information.

The key I generated does not have a password on it.

The title property can have spaces right?

I'd love to get this thing working, and it'd be awesome if this is just some stupid error I've made. Thanks for looking into it.

**Comment by NGPixel** [comment]
Which OS and version?
Which version of Git is installed? `git --version`
Paste the contents of the git section of your config.yml with the sensitive info redacted.

**Comment by SkyeRangerDelta** [comment]
OS: Linux Ubuntu 18.04.2 LTS
Git: 2.17.1

Config.yml attached (Sorry, the editing was being annoying)
![notepad++_2019-04-07_17-08-59](https://user-images.githubusercontent.com/12277949/55689846-db6ea880-5957-11e9-8b61-c464c08a5831.png)

**Comment by NGPixel** [comment]
The `url` is wrong. You're using an HTTPS formatted URL with SSH authentication.

The url should be `git@github.com:SkyeRangerDelta/YOUR_REPO.git`

**Comment by SkyeRangerDelta** [comment]
That finished the setup process. But now when it restarted it gave me a 403 permissions error. This is my end, what permissions does the folder for the wiki need (or do you recommend)? 755?

**Comment by SkyeRangerDelta** [comment]
I've set up a test environment away from my actual apache webserver to find out what I'm not doing right and I've figured out the permissions problems causing the 403 but now there's no CSS/Styling. I've added the port I'm connecting to in my browser to the host and port values in the config.yml and it's still not showing properly. Switching the port value back to 80 seems to never even load the wiki.

Since I'm running wiki.js in a subdomain folder (root folder of the apache virtual host), do I need to reconfigure something? I've experimented with the problem resolution as Issue 8 here: https://github.com/Requarks/wiki/issues/8 and I'm still getting the same problem while using ProxyPass.
